### PR TITLE
chore: Load composer plugins from source in dev mode

### PR DESCRIPTION
**Pull request by dmaretskyi**
### Motivation / Background

This Pull Request ...

### CODE REVIEW

File: packages/apps/composer-app/tsconfig.json [hunk 1]
```diff
@@ -1,202 +1,191 @@
...
     {
-      "path": "../plugins/plugin-thread"
+      "path": "../plugins/plugin-thread",
     },
     {
-      "path": "../plugins/plugin-wildcard"
+      "path": "../plugins/plugin-wildcard",
     },
     {
-      "path": "../types"
```
Comment (wittjosiah) [inline packages/apps/composer-app/tsconfig.json:199]: trailing commas valid json?

File: toolbox.json [hunk 2]
```diff
@@ -21,6 +21,11 @@
     "fixedKeys": [
       "extends",
       "compilerOptions"
-    ]
+    ],
+    "pathMapping": {
+      "include": [
+        "packages/apps/plugins/*"
```
Comment (wittjosiah) [inline toolbox.json:27]: if I understand correctly, this means only the plugins will hot reload? lower level libraries will still require recompilation unless they're added to this list?

Comment (dmaretskyi) [reply toolbox.json:27]: yea, we can add them progressivelly as long as they are compiled "normally" (without any special plugins)

Comment (wittjosiah) [reply toolbox.json:27]: makes sense 👍 just wanted to test my understanding. In the long run lower level libs (e.g. protocols, dxos-ui, etc.) will be coming from another repo so this seems like the right level to set this at. Lower levels we should be test with unit tests/storybooks vs hotreloading composer

**Pull request merged**